scripts/python/packages/parthenon_performance_app/parthenon_performance_app/githubapp.py

```diff
@@ -580,11 +580,16 @@ def cloneWikiRepo(self):
             # Ensure local branches are synchronized with server
             g.execute(['git', 'fetch'])
             # Will not overwrite files but will reset the index to match with the remote
-            g.execute(['git', 'reset','--mixed','origin/master'])
+            if wiki_state == "hard":
+              g.execute(['git', 'reset','--hard','origin/master'])
+            elif wiki_state == "mixed":
+              g.execute(['git', 'reset','--mixed','origin/master'])
+            elif wiki_state == "soft":
+              g.execute(['git', 'reset','--soft','origin/master'])
```

Comment:
What if `wiki_state` is something unexpected? Should there be a default choice or error/warning message?

Reply:
That's a fair point, good catch!